feature/consent/src/main/java/com/simprints/feature/consent/screens/consent/helpers/ParentalConsentTextHelper.kt

```diff
@@ -117,4 +117,12 @@ internal class ParentalConsentTextHelper @Inject constructor(
         Modality.FINGERPRINT -> context.getString(R.string.consent_biometrics_access_fingerprint)
         else -> ""
     }
+
+    private fun StringBuilder.appendSentence(text: String): StringBuilder {
```

Comment:
Let's avoid the code duplication and move it to a single place of declaration

Reply:
Sure - see https://github.com/Simprints/Android-Simprints-ID/pull/568